src/components/Registration/validate.ts

```diff
@@ -0,0 +1,29 @@
+import * as Yup from "yup";
+
+ const lowercaseRegex = /(?=.*[a-z])/;
+ const uppercaseRegex = /(?=.*[A-Z])/;
+ const numericRegex = /(?=.*[0-9])/;
+
+ export const SignupSchema = Yup.object().shape({
```

Comment:
to przerobić na funkcję SignupSchama = (t) => Yup.object()...

Reply:
przy okazji chyba można też zamiast t("key") napisać t`key`

Reply:
próbowałam, błędów nie ma, ale walidacja yupa całkowicie przestaje działać :(